app/views/accounts/_pii.html.erb

```diff
@@ -60,7 +60,7 @@
 <% unless locked_for_session %>
   <div class="grid-row bg-base-lightest padding-x-2 padding-y-1 fs-12p">
     <div class="grid-col-12">
-      <%= image_tag asset_url('lock.svg'), width: 8, height: 10, class: 'margin-right-1' %>
+      <%= image_tag asset_url('lock.svg'), width: 8, height: 10, class: 'margin-right-1', alt: '', aria: { hidden: true } %>
       <%= t('account.security.text') %>
```

Comment:
Also described in the extended commit description, but this is decorative since ["the information provided by the image [is] given using adjacent text"](https://www.w3.org/WAI/tutorials/images/decorative/) ("Your profile information is locked for your security").